File: C/Ex11.c
#include<stdio.h>
#include<stdlib.h>

//Função principal do programa
int main(int argc, char const *argv[])
{
// Declaração das variáveis
float n1,n2;

// PROGRAMA 11 - MOSTRAR O NÚMERO MAIOR, SE EXISTIR, CASO CONTRÁRIO, ENVIAR MENSAGEM NºS IDÊNTICOS

//Exibe na tela 
printf("Programa para exibir o maior numero, se existir");

//Quebra de linha
printf("\n");

//Solicita entrada de dados para o usuário e grava o valor digitado na variável "n1"
printf("Digite o primeiro valor: ");
scanf("%f",&n1);

//Solicita entrada de dados para o usuário e grava o valor digitado na variável "n2"
printf("Digite o segundo valor: ");
scanf("%f",&n2);

if(n1>n2){
    printf("O maior valor e: %1.f",n1);
}
else if(n1<n2){
    printf("O maior valor e: %1.f",n2);
}
else{
    printf("Os numeros sao identicos");
}
return 0;

}

File: PYTHON/Ex11.py
# Mostrar o maior entre 2 números, se existir, caso contrário, mostrar números idênticos

n1 = float(input('Digite o primeiro número: '))
n2 = float(input('Digite o segundo número: '))

if n1>n2:
    print(f"O maior número é: {n1:.2f}")
elif n1<n2:
    print(f"O maior número é: {n2:.2f}")
else:
    print(f"Os números são idênticos")